src/schema/v2/filterArtworksConnection.ts

```diff
@@ -497,6 +497,8 @@ export function filterArtworksConnection(primaryKey?: string) {
   )
 }
 
+export default filterArtworksConnection
```

Comment:
We should get rid of default exports at any opportunity. Let's remove this in the follow-up.

FIXME (so I can search this page easily)

Reply:
I specifically re-added this because there was some code dynamically importing it and checking for defaults...

I feel like it might've been the object_identification file, but I can't recall 100%. 

https://github.com/artsy/metaphysics/blob/9ded8d0410f4f48c363cccd8d712f5a95b061568/src/schema/v2/object_identification.ts#L80

Reply:
Gotcha